.eslintrc

```diff
@@ -9,6 +9,7 @@
     "console": true,
     "module": true,
     "require": true,
+    "window": true
```

Comment:
Note the comment above: it says that if additional globals are required, they should be added per-file -- see http://eslint.org/docs/user-guide/configuring.html#specifying-globals.